### ISSUE TITLE
Contract award data – number of bids

### ISSUE THREAD

**Issue by hutershvili** [digiwhist/opentender-frontend#3]
I was wondering: Is there a way to filter contract award data for a specific country or buyer by the number of bids, i.e. to identify contract awards that only had 1 bidder (to the extent that the data is available)? Is it possible to download contract award data that includes the number of bids?

Thanks!

**Comment by ffalt** [comment]
In the data, tender lots do have a bidsCount number field. It is not in the filters right now. Many entries lack it, when the information was not available.

So, yes, download with included number of bids is possible, but it depends heavily on country/buyer/data source combination.

**Comment by hutershvili** [comment]
For me, it would be very helpful to have the bidsCount as a filter option
on the search page (and the option to filter for empty ones):
even where no data was released, that fact is useful information – it might
serve as a red flag for users to follow up through a request for
information, for example.


2018-01-29 11:01 GMT+01:00 ffalt <notifications@github.com>:

**Comment by ffalt** [comment]
Yes. in fact, as red flag, the "Single Bidder Contract" Indicator is part of the Integrity Score, and it's based on bidsCount, see https://docs.google.com/spreadsheets/d/1ARtS9tM0jOfjCgvAbHKReV1lrrl7yNHDUoEONyAM7fY 
I'm adding a search filter for it to the wishlist.
